<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>renamedialog</class>
 <widget class="QDialog" name="renamedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rename Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set New Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="variableButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Variable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="templatesButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Templates</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="umdlist"/>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectallButton">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectNoneButton">
        <property name="text">
         <string>Select None</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>251</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>selectallButton</sender>
   <signal>pressed()</signal>
   <receiver>renamedialog</receiver>
   <slot>selectallAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectNoneButton</sender>
   <signal>pressed()</signal>
   <receiver>renamedialog</receiver>
   <slot>selectnoneAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>pressed()</signal>
   <receiver>renamedialog</receiver>
   <slot>startAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectallAction()</slot>
  <slot>selectnoneAction()</slot>
  <slot>startAction()</slot>
 </slots>
</ui>
